### DISCUSSION TITLE
Icon Request: Power Icon

### DISCUSSION THREAD

**Discussion by ashleyisles** [tailwindlabs/heroicons#407] [Ideas]
Love all of these icons! From what I've seen, there isn't a power icon, and I thought that might be a good add! Something like this -

![image](https://user-images.githubusercontent.com/6678189/90083800-254c6900-dcd9-11ea-8828-4a86804d62b0.png)

**Comment by jams77** [comment]
> Love all of these icons! From what I've seen, there isn't a power icon, and I thought that might be a good add! Something like this -
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/6678189/90083800-254c6900-dcd9-11ea-8828-4a86804d62b0.png)

Hey that's a great add icon and what a way to catch it!!!
